.env

```diff
@@ -4,7 +4,7 @@ STARDOG_USER=admin
 STARDOG_PASS=admin
 
 # Zookeeper version tag
```

Comment:
notice that line 2:
```
STARDOG_TAG=latest
```
it's irrelevant as the main image is being pulled from https://github.com/stardog-union/pystardog/blob/d5d11e6571091c03f1348e2711a830abf28b42b4/dockerfiles/dockerfile-stardog#L10
which is not taking any argument, hence always pulling the latest. adding an ARG here won't make much, since the image is being pulled from dockerhub, but we don't really push older versions to dockerhub, we just keep overwriting `latest`